### PR TITLE
Cleanup: renaming and ripple

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

**Comment by rhoads-zach** [comment]
@ALRubinger @edewit mind taking a look?

**Comment by rhoads-zach** [comment]
@inoxx03 mind taking a look?

File: docs/topics/getting-started-launchpad-create-runtime-project-framework.adoc [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 . Select _Begin by selecting a runtime project framework_ and click _Next_.
 . Choose the project type you would like to run. You may update any of the pre-filled details or leave them with the default details. Click _Next_.
 . Fill in the setup options and choose any dependencies you would like to have preconfigured. Click _Finish_
 
-NOTE: You can come back to the Launchpad later and try different options. 
+NOTE: You can come back to the {launcher} application later and try different options. 
 
 
-You have just created your runtime project framework project are now on the launch screen. Follow the steps in xref:launchpad-launch-runtime-project-framework[] to deploy it.
+You have just created your runtime project framework project are now on the launch screen. Follow the steps in xref:launcher-launch-runtime-project-framework[] to deploy it.
```
Comment (inoxx03) [inline docs/topics/getting-started-launchpad-create-runtime-project-framework.adoc]: There seems to be a word missing from this sentence.

**Comment by inoxx03** [comment]
Only one comment. Otherwise, LGTM:)

File: docs/topics/getting-started-launchpad-launch-booster.adoc [hunk 2]
```diff
@@ -12,7 +12,7 @@ This will:
 
 * Create a new GitHub repository associated with your account containing the mission's code.
 * Create a pipeline to continuously deliver your mission's code.
-* The pipeline will then build and deploy your mission's code to {OpenShiftLocal}.  The first launch and deployment should take under 10 minutes on most systems (depending upon resources and internet connection) as the Docker images powering the build are downloaded and cached. While the Docker images are being downloaded, you will see `Jenkins doesn’t have label launchpad-maven`; that's OK. Subsequent runs will be faster.
+* The pipeline will then build and deploy your mission's code to {OpenShiftLocal}.  The first launch and deployment should take under 10 minutes on most systems (depending upon resources and internet connection) as the Docker images powering the build are downloaded and cached. While the Docker images are being downloaded, you will see _Jenkins doesn’t have label..._ , which is expected. Subsequent runs will be faster.
```
Comment (Ladicek) [inline docs/topics/getting-started-launchpad-launch-booster.adoc]: TODO label name

Comment (rhoads-zach) [inline docs/topics/getting-started-launchpad-launch-booster.adoc]: @Ladicek I removed the label name to allow for it to change going forward.

Comment (Ladicek) [inline docs/topics/getting-started-launchpad-launch-booster.adoc]: Got it, but it looks incomplete like this. If you're aware of that, then fine :-)

Comment (rhoads-zach) [inline docs/topics/getting-started-launchpad-launch-booster.adoc]: @ALRubinger are you fine with this approach as it follows with the renaming work?

File: docs/topics/getting-started-launchpad-create-runtime-project-framework.adoc (hunk 1 above)
Comment (rhoads-zach) [inline docs/topics/getting-started-launchpad-create-runtime-project-framework.adoc]: thank you @inoxx03, fixed

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/2584a07351c9c336fb59ac6ec52d799f22031c0b

**Pull request closed without merging**